CONTRIBUTORS.txt

```diff
@@ -1,4 +1,4 @@
-﻿Contributors
+Contributors
```

Comment:
There's a modification here, I don't know why ;)

Reply:
This is just a guess but I think my editor ate the byte order mark?

Reply:
That was it. Restored :)